calcom/cal.com
# fix: DST issue in the month of DST change

## What does this PR do?

Fixes issue that availability was shown wrong in month of dst change. 

Fixes #10517

## Type of change


## How should this be tested?

- Create availability in NY timezone 
- Go to booking page also set NY timezone
- Go to November. All days should start at the time defined in the availability (before and after 5th of november)
- Change time to Europe/Amsterdam on booking page
- See that on oktober 28 availability starts on defined time but after oktober 28 until nov 5 availability starts 1 hour earlier 
- Test other timezones

## Mandatory Tasks
